src/Console/Command/ModuleRegisterCommand.php

```diff
@@ -0,0 +1,46 @@
+<?php
+declare(strict_types=1);
+
+namespace LotGD\Core\Console\Command;
+
+use LotGD\Core\Bootstrap;
```

Comment:
I would say we use first external libraries grouped by the library, then internal libraries.